### ISSUE TITLE
Suggestion: Link search function from vocabulary homepage

### ISSUE THREAD

**Issue by callumrollo** [nvs-vocabs/L35#2]
I find the search link in the README very useful! https://vocab.nerc.ac.uk/search_nvs/L35/

Previously I was just searching on the page with CTRL-F, which is quite inefficient. I think it would be good to link directly to the search page for L35 from the homepage for L35 ie the webpage http://vocab.nerc.ac.uk/collection/L35/current/ should feature the link to https://vocab.nerc.ac.uk/search_nvs/L35/ prominently. The fact that this repo features both these links in the README suggests that this would be desirable, presumably for all of the NVS pages.

Currently you have to click the generic "Search NVS" link, then find use the quite unintuitive dropdown "vocab id" and select L35 from many options to search just within the vocabulary L35

Maybe a "search this vocabulary" link within the empty whitespace here would work?

![image](https://github.com/user-attachments/assets/b604c933-24b2-41cf-9ab1-1f661f71d883)

**Comment by danibodc** [comment]
Thanks for the feedback @callumrollo, after you click 'Search NVS' you are taken to https://vocab.nerc.ac.uk/search_nvs/. From here there is an option to search for a particular vocabulary - if you were to type 'L35' in to the box below 'Search for vocabulary collections', you are then directed towards the link you mentioned: https://vocab.nerc.ac.uk/search_nvs/L35/.

![image](https://github.com/user-attachments/assets/4cbf1cdf-1243-4888-b798-4c0eab01ad55)

I agree if you have already navigated to http://vocab.nerc.ac.uk/collection/L35/current/ it would be useful to have the search box here, or have a 'Search' button that takes you to https://vocab.nerc.ac.uk/search_nvs/L35/. So can certainly raise this.

**Comment by danibodc** [comment]
Alternatively if you type (or select from the drop-down) L35 in the 'Vocab ID box' under 'Search for a term in a vocabulary collection' at https://vocab.nerc.ac.uk/search_nvs/, this then presents the whole collection with a search box:

![image](https://github.com/user-attachments/assets/e84966fa-11fa-4d06-a72c-cc3f7cfcdef1)

![image](https://github.com/user-attachments/assets/c75faa30-9c81-49f8-80a9-98f51c9b0905)

**Comment by callumrollo** [comment]
Yes! I make use of the search page now, but when I started using the vocabularies as a very non-expert user, it wasn't clear to me how to just search the vocabulary I was interested in (e.g. L35), so I defaulted to CTRL-F. I think a link to the search page for the collection in the block at the top of the collection page would improve usability of the service
